.travis.yml

```diff
@@ -115,7 +120,7 @@ jobs:
     # -------------------------------------------------------------------------
     - stage: Deployment
       name: Deploy to next-docs.kuzzle.io
-      if: branch = 3-dev
+      if: branch =~ /[0-9]+-dev$/
```

Comment:
Bad idea for me. It means that if someone (I don't know why) perform a commit on 2-dev branch for example, this deployment is triggered. :slightly_smiling_face:

Reply:
We should always specify branch we want to deploy IMH

Reply:
oh yes ur right!